src/modules/Exposure/info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Exposure",
+  "description": "Change the exposure of the image by given exposure value",
+  "inputs": {
+      "exposure": {
+          "type": "range",
```

Comment:
You can make the type as float as it will be mapped to range in the ui automatically.